user/decorator.py

```diff
@@ -0,0 +1,26 @@
+import bcrypt
+import jwt 
+
+from django.http           import JsonResponse, HttpResponse
+
+from devx.settings         import SECRET_KEY
+from devx.settings         import ALGORITHM
+from .models               import User
+
+def login_decorator(func):
+    def wrapper(self,request,*args,**kwargs):
+        try:
+            access_token    = request.headers.get('Authorization',None)
+            
+            if access_token :                 
+                payload         = jwt.decode(access_token, SECRET_KEY, algorithms= 'ALGORITHM')
```

Comment:
```suggestion
                payload         = jwt.decode(access_token, SECRET_KEY, algorithms = 'ALGORITHM')
```
지연님 이런 띄어쓰기, 간격이 규칙적으로 동일해야 합니다. 이런 작은 디테일이 큰 가독성을 만들고 코드의 질을 좌우합니다. 조금씩 더 신경써주세용 👍

Reply:
넵 코드의 질을 위해 앞으로 신경쓰겠습니다!!